packages/views/organization_form.jade

```diff
@@ -5,11 +5,12 @@ template(name="organizationForm")
         .row
           .col-sm-12
             label Name
-              input#organization-name.form-control(name="name" type="string")
+              .help-inline= errorForName
+              input(required='true')#organization-name.form-control(name="name" type="string")
```

Comment:
organization_form.jade, line 9:s             /
You should not have jade tags with multiple attributes.
organization_form.jade, line 13:
You should not have jade tags with multiple attributes.